Kickstarter-iOS/Views/Controllers/LoginToutViewController.swift

```diff
@@ -115,6 +117,17 @@ internal final class LoginToutViewController: UIViewController, MFMailComposeVie
         Strings.login_tout_back_intent_traditional_login_button()
       }
 
+    _ = self.fbDisclaimerStackView
+      |> baseStackViewStyle
+      |> UIStackView.lens.spacing .~ Styles.gridHalf(1)
+
+    _ = self.getNotifiedLabel
+      |> baseLabelStyle
+      |> UILabel.lens.font %~~ { _, label in
+      label.traitCollection.isRegularRegular ? .ksr_headline(size: 15.0) : .ksr_headline(size: 12.0)
```

Comment:
Opted for `headline` font here because `caption1` caused it wrap due to trucation.

![Simulator Screen Shot - iPhone Xs - 2019-10-08 at 15 03 18](https://user-images.githubusercontent.com/11362005/66428464-12477900-e9e4-11e9-94eb-fbf03f94d4e1.png)